packages/common/utils/item.js

```diff
@@ -18,3 +18,21 @@ export function checkItemsAfterDecryption(items) {
     [],
   );
 }
+
+export function generateSystemItemName(teamId) {
+  return `team-${teamId}`;
+}
+
+export function generateSystemItemEmail(teamId) {
+  return `${generateSystemItemName(teamId)}@${location.hostname}`;
```

Comment:
Need to wrap the global vars in to a function.

Reply:
Done